newIDE/app/src/UI/MarkdownText.js

```diff
@@ -44,7 +49,15 @@ const makeMarkdownCustomComponents = (
       return null;
     }
     return isStandaloneText || allowParagraphs ? (
-      <p>{props.children}</p>
+      shouldFakeListItem ? (
+        <ul className="faked-list-item">
```

Comment:
it's not fake technically, more _generated_ ?
Should it be "generated-list-item" ?